include/gz/launch/config.hh.in

```diff
@@ -17,3 +17,8 @@
 #cmakedefine BUILD_TYPE_PROFILE 1
 #cmakedefine BUILD_TYPE_DEBUG 1
 #cmakedefine BUILD_TYPE_RELEASE 1
+
+namespace [[deprecated("Use gz namespace.")]]  ignition
```

Comment:
We should make sure that the deprecated header includes `config.hh`.

Reply:
https://github.com/gazebosim/gz-launch/pull/163/commits/5d0d44f067571dabf506493e8c1961e376c0d804